.circleci/config.yml

```diff
@@ -0,0 +1,116 @@
+version: 2
+jobs:
+  build:
+    machine: true
```

Comment:
We've been using workflows with separate images (in particular, testing on different Node versions, just like Travis does by default) and it's been working well. Not sure about performance disadvantages though.